### DISCUSSION TITLE
SABnzbd and Mylar3 stopped working

### DISCUSSION THREAD

**Discussion by dltyih** [OneCDOnly/sherpa#286] [General]
Can anyone help? SABnzbd and Mylar3 suddenly stopped working and remains "inactive" despite being enabled. Tried starting / activating, reinstalling, and uninstall and then install. Sherpa says For more information: /etc/init.d/sabnzbd.sh log. I followed the path and sabnzbd.sh does not appear to be a log? Many thanks

**Comment by OneCDOnly** [comment]
When you run `/etc/init.d/sabnzbd.sh log`, the service-script loads a log file from `/var/log/SABnzbd.log` because it's a command to run, not the file to load. ;)

**Reply by dltyih** [reply]
Many thanks!

I have attached the log file. Seems there is a failure at "create new virtual Python environment". I am no good with diagnosing where the problem came up. Would you be able to point me to the problem? Many thanks!
[SABnzbd.log](https://github.com/OneCDOnly/sherpa/files/14427643/SABnzbd.log)

**Comment by OneCDOnly** [comment]
Your log shows the Python module for `virtualenv` can't be found. I think your Python environment is messed-up. 

Can you please reinstall Entware?
```
sherpa reinstall ew
```

**Reply by dltyih** [reply]
Thank you so much! It worked for SABnzbd, and luckily I have a backup from a week ago, so everything restored. Very lucky. Thank you!

Unfortunately, Mylar3 does not work. Log file attached. Seems to be something entirely different now ... and this is way over my head. Would you help point me in the right direction? I even tried uninstalling and then installing Mylar3 again. Many thanks!
[Mylar3.log](https://github.com/OneCDOnly/sherpa/files/14431896/Mylar3.log)

**Comment by OneCDOnly** [comment]
Fixed https://github.com/OneCDOnly/sherpa/issues/287
